### PR TITLE
Used logger rather than prints

### LINKED ISSUE #1110: Use logger rather than prints
Many modules still have prints though it would be preferable to use the logger instead to give more control to the users what content they want to see.

Triggered by this #1093 PR, but it doesn't only affect the tap module.

### PR DESCRIPTION
**Pull request by Phantsure**
Fixes #1110 
Changelog:
- Added log for utils/tap/core.py, lcogt/core.py, oac/core.py

### CODE REVIEW

**Comment by bsipocz** [comment]
@Phantsure - do you plan to fix similar instances in other modules, too in this PR?

**Comment by Phantsure** [comment]
Plan is to fix for other modules, might start today. Sorry not replying, due to assignments and traveling

**Comment by Phantsure** [comment]
should this line be changed?
https://github.com/Phantsure/astroquery/blob/7534ca04f1ea646d165ae578f0e68ebf07aa0d4c/astroquery/alma/core.py#L785

**Comment by bsipocz** [comment]
Good point, I think those either need to be a warning or a raised exception rather than a print (depending on whether any functionality could work without mocpy/regions or not).

**Comment by Phantsure** [comment]
so this should be a separate issue or should it be just a warning?
and how could i remove this checks error(codecov/patch)?

File: astroquery/oac/core.py [hunk 1]
```diff
@@ -488,12 +488,12 @@ def _parse_result(self, response, verbose=False):
             output_response = self._format_output(raw_output)
 
         except AttributeError:
-            log.debug("ERROR: The web service returned error code: %s" %
+            log.info("ERROR: The web service returned error code: %s" %
                   response.status_code)
             return
```
Comment (bsipocz) [inline astroquery/oac/core.py]: I'm not sure why this, or the KeyError below are not raised. @guillochon or @pcowpert could you please comment on it?

Comment (guillochon) [reply astroquery/oac/core.py]: These do look like they should be `raises` rather than `returns` @bsipocz. Fine to change them to be I think.

Comment (Phantsure) [reply astroquery/oac/core.py]: So should i `raise` or make a new issue for this?

Comment (bsipocz) [reply astroquery/oac/core.py]: better change it in this PR.

Comment (Phantsure) [reply astroquery/oac/core.py]: Done

File: astroquery/oac/core.py [hunk 2]
```diff
@@ -479,22 +479,20 @@ def _parse_result(self, response, verbose=False):
 
         try:
```
Comment (bsipocz) [inline astroquery/oac/core.py]: there is no need for this try except any more, please remove.

Comment (Phantsure) [reply astroquery/oac/core.py]: Done

**Comment by cbanek** [comment]
I'm willing to help out with this one.  Logging library is definitely the way to go!

File: astroquery/oac/core.py [hunk 3]
```diff
@@ -476,25 +477,15 @@ def _parse_result(self, response, verbose=False):
         if not verbose:
             commons.suppress_vo_warnings()
 
-        try:
-            if response.status_code != 200:
-                raise AttributeError
+        if response.status_code != 200:
```
Comment (bsipocz) [inline astroquery/oac/core.py]: We could simply use `response.raise_for_status()` as we do in the other modules.

**Comment by bsipocz** [comment]
I go ahead and merge this now, and will open a separate issue for checking the whole codebase for cases where `response.raise_for_status()` could be used instead of the hacky ways of checking the status code manually.

**Comment by bsipocz** [comment]
Thanks @Phantsure!

**Pull request merged**